net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/bgp.volt

```diff
@@ -72,6 +76,61 @@ POSSIBILITY OF SUCH DAMAGE.
             </tfoot>
         </table>
     </div>
+                
+    <div id="aspaths" class="tab-pane fade in">
+        <table id="grid-aspaths" class="table table-responsive" data-editDialog="DialogEditBGPASPaths">
+            <thead>
+                <tr>
+                    <th data-column-id="enabled" data-type="string" data-formatter="rowtoggle" data-sortable="false">{{ lang._('Enabled') }}</th>
+                    <th data-column-id="number" data-type="string" data-visible="true" data-sortable="true">{{ lang._('Number') }}</th>
+                    <th data-column-id="action" data-type="string" data-visible="true" data-sortable="false">{{ lang._('Action') }}</th>
+                    <th data-column-id="as" data-type="string" data-visible="true" data-sortable="false">{{ lang._('AS') }}</th>
```

Comment:
I would write the string in full length (expecting autonomous system for AS here, but a translator cannot know that because the context is lost)